engine/apps/mobile_app/views.py

```diff
@@ -197,6 +197,9 @@ def _proxy_request(self, request: Request, *args, **kwargs) -> Response:
         downstream_path = kwargs["downstream_path"]
         method = request.method
         user = request.user
+        org = user.organization
+        if not org.is_grafana_incident_enabled:
```

Comment:
_nit_ this is probably fine for now. Eventually might make sense to move this to a separate method, something like `can_proxy_request_to_downstream_backend`, which has different conditional checks based on the the value of `kwargs["downstream_backend"]` (ie. `incident`, `slo`, etc). For now we only support Incident so this is okay.. just thought I'd note.